Add support for JOIN USING statements (#9629)
crate/crate: docs/appendices/release-notes/unreleased.rst

Changes
=======

> - Added support for ``JOIN USING``, e.g. select * from t1 join t2 using(col1, col2),

Reviewer: ```suggestion
- Added support for ``JOIN USING``, e.g. ``SELECT * FROM t1 JOIN t2 USING (col)``,
```

>   a more convenient alternative to ``JOIN ON``, when we know both tables have the

Reviewer: ```suggestion
  an alternative to ``JOIN ON``, when the column name(s) are the same in both relations.
```
